src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -446,7 +484,11 @@ class Manager : public DoutPrefixProvider {
       // log when queue is not idle
       ldpp_dout(this, 20) << "INFO: found: " << total_entries << " entries in: " << queue_name <<
         ". end marker is: " << end_marker << dendl;
-      
+      RGWPubSubEndpoint::Ptr push_endpoint;
+      if (create_push_endpoint(queue_name, entries, yield, &push_endpoint)
+          < 0) {
+        return;
```

Comment:
this means that you stop processing of the queue. i think we should continue queue processing unless we lose ownership of queue is removed.
this will alllow users to fix conf errors

Reply:
if you exit from the function, nothing will reschedule the processing of the queue and the rgw will release ownership.
handling should be similar to the other error cases (="continue"), and wait for the problem to be resolved.
note that we will not be scheduling the coroutine too often in this case, with the timer we have in line 426.

Reply:
done